sbnanaobj/StandardRecord/SRNuGraphScore.h

```diff
@@ -0,0 +1,33 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRNuGraphScore.h
+////////////////////////////////////////////////////////////////////////
+#ifndef SRNuGraphSCORE_H
+#define SRNuGraphSCORE_H
+
+namespace caf {
+
+  class SRNuGraphScore {
+  public:
+    SRNuGraphScore();
+    ~SRNuGraphScore() {}
+
+    enum NuGraphCategory {
+      Unset = -1,
+      Mip = 0,
+      Hip = 1,
+      Shower = 2,
+      Michel = 3,
+      Diffuse = 4
```

Comment:
Shouldn't MIP and HIP go all capital?
Also have some comment, e.g. to explain that MIP and HIP refer to a track-like object, what is "diffuse" meaning, that Michel is not the [pianist](https://en.wikipedia.org/wiki/Michel_Petrucciani)...
```suggestion
      Unset = -1,  ///< Unset/unknown/undecided/ category.
      Mip = 0,     ///< Track from a minimum-ionizing particle.
      Hip = 1,     ///< Track from a high ionization particle.
      Shower = 2,  ///< Particle shower.
      Michel = 3,  ///< Michel electron (from muon decay).
      Diffuse = 4  ///< Diffuse background.
```
Still not sure how to define `Diffuse`.